app/setup.py

```diff
@@ -25,9 +25,9 @@
 from app.keys import KEY_PURPOSE_SUBMISSION
 from app.helpers import get_span_and_trace
 from app.secrets import SecretStore, validate_required_secrets
-from app.storage.datastore import DatastoreStorage
-from app.storage.dynamodb import DynamodbStorage
-from app.storage.redis import RedisStorage
+from app.storage.datastore import Datastore
```

Comment:
We can import these classes into `app.storage.__init__.py` and import them from there